db/index.js

```diff
@@ -27,7 +27,7 @@ const repos = require('./repos');
 // TODO: Had to change below from 'const' into 'let' because of the nasty 'rewire' hacks inside DBSandbox.js.
 // eslint-disable-next-line prefer-const
 let initOptions = {
-	pgNative: true,
```

Comment:
default is `false`, we can remove entirely

Reply:
I'd prefer to leave it until we start QA testing.

Reply:
kind of pointless, because we removed dependency, but as you wish ;)